adopt-counter/index.js

```diff
@@ -0,0 +1,66 @@
+const request = require('sync-request')
+const {
+  Client
+} = require('pg')
+const client = new Client({
+  user: 'postgres',
```

Comment:
Is it common in Node land to externalize all of those constants or put them at the top as JSON or????..